main/test/ca/uwaterloo/flix/library/Core/Io/TestInputStream.flix

```diff
@@ -0,0 +1,144 @@
+use Core/Io/InputStream.{available, close, fromBuffer, InputStream, readByte, readBuffer, readBufferWithLength};
+use Core/Io/InputStream.InputStream.InputStream;
+use Core/Io/IOError.IOError;
+
+namespace Core/Io/TestInputStream {
+    def testBuffer(): InputStream & Impure = fromBuffer([0 as Int8, 1 as Int8, 2 as Int8])
```

Comment:
In our test files, all @test definitions goes first and then we have the helper functions at the bottom.